src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java

```diff
@@ -64,8 +65,8 @@ public void lock() {
             long start = System.currentTimeMillis();
             while (!lock.compareAndSwapValue(UNLOCKED, PID)) {
                 // add a tiny delay
-                if (Thread.interrupted())
-                    throw new IllegalStateException("Interrupted for the lock file:" + path);
+                if (Thread.currentThread().isInterrupted())
```

Comment:
should we `Thread.currentThread().interrupt(); ?`

Reply:
No
> The interrupted status of the thread is unaffected by this method.
From https://docs.oracle.com/javase/8/docs/api/java/lang/Thread.html#isInterrupted--

Reply:
Oh yes of course. It is friday afternoon after all ;)